src/fiatconnect/saga.ts

```diff
@@ -1002,13 +1003,25 @@ export function* handleCreateFiatConnectTransfer(
   const quoteId = fiatConnectQuote.getQuoteId()
   let transactionHash: string | null = null
   try {
-    const { transferAddress }: TransferResponse = yield call(_initiateTransferWithProvider, action)
+    const { transferAddress, transferId }: TransferResponse = yield call(
+      _initiateTransferWithProvider,
+      action
+    )
 
     if (flow === CICOFlow.CashOut) {
       transactionHash = yield call(_initiateSendTxToProvider, {
         transferAddress,
         fiatConnectQuote,
       })
+      yield put(
+        cacheFiatConnectTransfer({
+          txHash: transactionHash!.toLowerCase(),
```

Comment:
I guess `_initiateSendTxToProvider` guarantees returning a string, but maybe safer to just do an additional null check here and log a warning/error and skip caching transfer if its null?

Or you can also use a local variable to avoid the `!` operator. E.g.,
```js
const cashOutTxHash: string = yield call(...)
yield put(cacheFiatConnectTransfer({txHash: cashOutTxHash.toLowerCase(), ...}))
transactionHash = cashOutTxHash
```